contracts/erc20/contracts/src/v06/IERC20TokenV06.sol

```diff
@@ -0,0 +1,95 @@
+/*
```

Comment:
Feels bad bumping up the major version when I've only upgraded a handful of the contracts.

This would also freeze this contract package from the context of its dependents. Lerna will choose to install this package from npm for them. So we could never make changes to the previous version from within the monorepo and have it be reflected in its dependents, because it no longer technically exists in the monorepo.

Reply:
Thanks, yeah that does sound kind of hectic. Another option could be to keep this in a branch separate from `development` until the transition is complete, similar to how we used to do for new versions of the Exchange. What do you think, would that be more or less cumbersome?

Reply:
idk, I think that worked for 3.0 because we weren't immediately productionizing the changes. We'd be folding this stuff into asset-swapper and other packages pretty much right away so we'd be looking for a solution in 2 weeks again.